doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -36,6 +36,7 @@ In the current version, high-availability {CENTRAL} functionality is for Technol
 ** `rhdm-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} replicated pods that build a service from source. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image and displace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment. You can use any container-based integration workflows and do not need to use any other tools to manage the pods.
 endif::DM[]
 +
+. If you want to enable automatic upgrades to new versions, select the *Enable Upgrades* box. If this box is selected, when a new minor or patch version of {PRODUCT} 7.x becomes available, the Operator automatically upgrades your deployment to this version. All services are preserved and normally remain available throughout the upgrade process.
```

Comment:
Minor upgrades require the user to opt-in separately. I've change this section accordingly... thoughts?
```
If you want to enable automatic upgrades to new versions, select the *Enable Upgrades* box. If this box is selected, when a new patch version of {PRODUCT} 7.x becomes available, the Operator automatically upgrades your deployment to this version. Select the "Include minor version upgrades" box as well if you want the same upgrade behavior for minor versions. All services are preserved and normally remain available throughout the upgrade process.
```